db/fixtures/property_specs/ems_inventory.yml

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
/db/fixtures feels like the wrong place because they are neither db related nor fixtures. Wonder if just /data is good enough.

Reply:
as discussed, config is even better.